README.md

```diff
@@ -1,8 +1,21 @@
 # maped
+
 Amstrad CPC (Plus/GX4000) tile map editor
 
 ![Screenshot of application](/../doc-assets/screenshot.png?raw=true)
 
 `maped` is a tile map editor specifically targetting the Amstrad CPC Plus. It has built-in support for the standard CPC screen modes and enforces restrictions that make sense given the platform. It is designed to be used in conjunction with an image editor and is most useful for editing existing image maps and assigning and keeping track of map metadata.
 
-`maped` depends on Python 3, pytk (distributed with Python by default on Windows and Mac) and pypng. pypng can be installed using pip, by running `pip install pypng` or equivalent from a terminal. Linux users may want/need to use their distribution's package manager to install dependencies.
+## Install and usage
+
+`maped` uses [poetry](https://python-poetry.org). Making use of `maped` requires `poetry` to be [installed in your system](https://python-poetry.org/docs/#installation). Once available, use the following commands:
```

Comment:
I like that the editor (at least at the moment) is very lightweight. Originally, I wanted to have no dependencies outside of what Python bundles, but didn't realise pypng wasn't part of that... I'd prefer to keep the original paragraph and add this as an extra paragraph that says something along the lines of "`maped` dependencies can optionally be managed by `poetry` (link). When available, ..."

Reply:
I partially restored the previous paragraph, including links to sources but omitting the installation of `pypng` via the system package. Let me know what you think of the new version

Reply:
Looks good :)